src/Raven.Client/Indexing/IndexDefinition.cs

```diff
@@ -69,46 +69,84 @@ public IndexConfiguration Configuration
             set { _configuration = value; }
         }
 
+        public IndexDefinitionCompareDifferences Compare(IndexDefinition other)
```

Comment:
I know, but it would be hard to implement this without touching the old client. I'm skipping my next two issues, waiting for the NewClient.